Containerfiles/centos9.Containerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:stream9 as base
+
```

Comment:
Actually, I don't think we should use CentOS Stream here. It might be better to use Oracle, Alma, or Rocky Linux 9

Reply:
My suggestion would be (now that Alma and Rocky conversions are enabled for the product offering) that we use Alma Linux 9 as base

Here is the one I setup for Oracle before
https://github.com/oamg/convert2rhel/blob/el9-conversions/Containerfiles/rpmbuild.oracle9.Containerfile

Reply:
Not sure about the choice here.

I can see how for integration or e2e testing we should better support the systems which are the targets for the conversion. But in case of unit testing - the important part here is not the OS but rather the version of the Python library and the base dependencies.

As these unit tests run at the development phase, and do not block the delivery of the release, here we would benefit from looking slightly ahead and more continuously integrating with CentOS Stream than to pin to certain snapshot of the already released system.

Reply:
True. Sounds like a good idea

We treated it as an early indication of the OS itself doing something unusual and just easier align with the systems we're converting. But this was established before integration tests were developed